modules/oe_content_event/modules/oe_content_event_event_programme/config/install/oe_content_event_event_programme.oe_event_programme_type.default.yml

```diff
@@ -0,0 +1,6 @@
+langcode: en
+status: true
+dependencies: {  }
+id: default
```

Comment:
Maybe better to use `oe_default` machine name for consistency with other corporate entity types?